tool/teleport/common/teleport.go

```diff
@@ -472,6 +472,7 @@ func Run(options Options) (app *kingpin.Application, executedCommand string, con
 	integrationConfDeployServiceCmd.Flag("aws-region", "AWS Region.").Required().StringVar(&ccf.IntegrationConfDeployServiceIAMArguments.Region)
 	integrationConfDeployServiceCmd.Flag("role", "The AWS Role used by the AWS OIDC Integration.").Required().StringVar(&ccf.IntegrationConfDeployServiceIAMArguments.Role)
 	integrationConfDeployServiceCmd.Flag("task-role", "The AWS Role to be used by the deployed service.").Required().StringVar(&ccf.IntegrationConfDeployServiceIAMArguments.TaskRole)
+	integrationConfDeployServiceCmd.Flag("aws-account", "The AWS account ID.").Required().StringVar(&ccf.IntegrationConfDeployServiceIAMArguments.AccountID)
```

Comment:
```suggestion
	integrationConfDeployServiceCmd.Flag("aws-account-id", "The AWS account ID.").Required().StringVar(&ccf.IntegrationConfDeployServiceIAMArguments.AccountID)
```
For consistency with
https://github.com/gravitational/teleport/blob/1531595d12e68139d5432af90a5dee5ada4e5324/tool/teleport/common/teleport.go#L286
and
https://github.com/gravitational/teleport/blob/1531595d12e68139d5432af90a5dee5ada4e5324/tool/teleport/common/teleport.go#L241